.changelog/unreleased/bug-fixes/1175-cw-consensus-state-height-iteration-codec.md

```diff
@@ -0,0 +1,4 @@
+- [ibc-clients/cw-context] Fix `ConsensusState` height iteration.
+  ([\#1175](https://github.com/cosmos/ibc-rs/issues/1175))
+- [ibc-clients/cw-context] Encode and decode `ConsensusState` heights using
+  Protobuf. ([\#1176](https://github.com/cosmos/ibc-rs/pull/1176))
```

Comment:
As this crate has recently been added (Not present in any versions yet), not sure if we should reflect this fix in the changelog.

Reply:
Ah I didn't know this convention. Removed.

7a107829

Reply:
7a10782